### ISSUE TITLE
could not find a declaration file for module 'venn.js' in React

### ISSUE THREAD

**Issue by indhumathys0** [benfred/venn.js#147]
Hi,

import venn from "venn.js" and import * as from venn from "venn.js" is working fine in React. But getting below error while project team members taking pull and doing npm install to import the dependencies in the project. we need to install separately "npm install venn.js" , then it's working fine. But in server environment it will load the dependencies and throwing the below error.please provide a solution ASAP,
<img width="535" alt="issue" src="https://user-images.githubusercontent.com/31129920/87254042-3ff2ae80-c49d-11ea-8e76-ab62cb0bada9.PNG">

**Comment by sgratzl** [comment]
I don't think that this version of the library comes with TypeScript typings. You could give my fork at https://github.com/upsetjs/venn.js a try

**Comment by indhumathys0** [comment]
> I don't think that this version of the library comes with TypeScript typings. You could give my fork at https://github.com/upsetjs/venn.js a try
